server_side/ArduinoRequest.py

```diff
@@ -0,0 +1,48 @@
+from server_side import Const
+from datetime import datetime
+
+
+def set_fall(data):
+    user_id = data.get(Const.USER_ID)
+    time = datetime.now()
+    data[Const.TIME] = time
+    loc = data.get(Const.FALL_LOC)
+    write_fall_to_db(user_id, time, loc)
+    # alert to app
+
+
+def set_heartbeat(data):
+    user_id = data.get(Const.USER_ID)
+    time = datetime.now()
+    val = data.get(Const.HEARTBEAT_VAL)
+    write_heartbeat_to_db(user_id, time, val)
+
+
+def set_anomaly_heartbeat(data):
+    user_id = data.get(Const.USER_ID)
+    time = datetime.now()
+    val = data.get(Const.HEARTBEAT_VAL)
+    write_heartbeat_to_db(user_id, time, val)
+    # alert to app
+
+
+def write_fall_to_db(user_id, fall_time, fall_location):
+    query = f"""
+    INSERT INTO {Const.FALL_TABLE} ({Const.USER_ID, Const.TIME, Const.FALL_LOC})
+    VALUES 
+       ({user_id},{fall_time}, {fall_location});
+    """
+    print(f"query was created: {query}")
```

Comment:
Consider using logger instead of print so you can write stuff to a log file as well as to the console print out, helping with later bug searches